cmd/client-s3.go

```diff
@@ -799,6 +800,23 @@ func (c *s3Client) Put(ctx context.Context, reader io.Reader, size int64, metada
 	if ok {
 		delete(metadata, "X-Amz-Storage-Class")
 	}
+
+	lockModeStr, ok := metadata["X-Amz-Object-Lock-Mode"]
```

Comment:
shall we define some global constants for these names?

Reply:
fixed